packages/office-ui-fabric-react/src/components/Button/Button.Props.ts

```diff
@@ -34,7 +35,7 @@ export interface IButtonProps extends React.HTMLAttributes<HTMLButtonElement | H
   /**
    * Custom styling for individual elements within the button DOM.
    */
-  styles?: IButtonStyles;
+  styles?: ISplitButtonStyles;
```

Comment:
I agree, this is weird. I'm not sure how to solve that issue though, since `SplitButtons` are being created with normal Buttons. 
Another option would be to create another `splitButtonStyles` prop, but this feels strange as well.